src/models/data_event.model.ts

```diff
@@ -0,0 +1,23 @@
+// DataEvent interface for the data event payload
+export interface DataEvent {
+  type: string,
+  name: string,
+  description: string,
+  timestamp: number,
+  timestamp_local: number,
+  tz_offset_minutes: number,
+  pluginId: number,
+  os: string,
+  version: string,
+  hostname: string,
+  timezone: string
+}
+
+export async function dataEventPayload(params: DataEvent) {
+  return {
+    schema: "iglu:com.software/code_event/jsonschema/1-0-0",
```

Comment:
i think we should match 1-1 with the snowlow event name. can we name this CodeEvent?

Reply:
yes, we should match it. sorry for the mismatch. I'll update it.

as we discussed in standup I'll change the entity to something related to dom events. DomEvent | BehaviorEvent | StateEvent ?

Reply:
oh and this one isn't related to the "code_event" entity. i think you were confused on that. it's actually called "data_event" in snowplow

Reply:
updated snowplow entity to dom_event and changed the names and schema reference in this as well